DFC.ServiceTaxonomy.Editor/Views/ContentVisualise.Button.cshtml

```diff
@@ -18,6 +18,18 @@
        asp-controller="Visualise"
        asp-area="@typeof(DFC.ServiceTaxonomy.GraphVisualiser.Startup)!.Namespace"
        asp-route-visualise="@graphSyncPartText"
+       asp-route-graph="published"
        target="_blank"
-       class="publish-button btn btn-info">@T["Visualise (last published)"]</a>
+       class="publish-button btn @(!string.IsNullOrWhiteSpace(Model.ButtonSize) ? "btn-sm" : "") btn-info">@T["Visualise (last published)"]</a>
```

Comment:
Could we use the passed ButtonSize and use that for the value after "btn-", rather than hard-coding "btn-sm".
Also, could the text be changed to "Visualise Published".

Reply:
Done!